### PR TITLE
Alerting: Update Kafka receiver to use encoding/json to build messages

### PR DESCRIPTION
**Pull request by yuri-tceretian**
**What is this feature?**
This PR refactors Kafka receiver to use structures for creating webhook messages instead of simplejson framework. This is a requirement for moving it to alerting repository.

### CODE REVIEW

File: pkg/services/ngalert/notifier/channels/kafka.go [hunk 1]
```diff
@@ -125,36 +147,36 @@ func (kn *KafkaNotifier) SendResolved() bool {
 }
 
 func (kn *KafkaNotifier) buildBody(ctx context.Context, tmpl func(string) string, as ...*types.Alert) (string, error) {
-	bodyJSON := simplejson.New()
-	bodyJSON.Set("client", "Grafana")
-	bodyJSON.Set("description", tmpl(kn.settings.Description))
-	bodyJSON.Set("details", tmpl(kn.settings.Details))
+	record := kafkaRecord{}
```
Comment (santihernandezc) [inline pkg/services/ngalert/notifier/channels/kafka.go:150]:
```suggestion
	var record kafkaRecord
```

Comment (santihernandezc) [reply pkg/services/ngalert/notifier/channels/kafka.go:150]: Adding `Client`, `Description` and `Details` in this same declaration could be an option too.

**Review by santihernandezc — APPROVED** [review]
LGTM 👍

File: pkg/services/ngalert/notifier/channels/kafka.go (hunk 1 above)
Comment (yuri-tceretian) [reply pkg/services/ngalert/notifier/channels/kafka.go:150]: the point was to change line-to-line

**Pull request merged**